fastchat/train/train.py

```diff
@@ -93,7 +118,9 @@ def preprocess(
         for j, sentence in enumerate(source):
             role = roles[sentence["from"]]
             assert role == conv.roles[j % 2], f"{i}"
-            conv.append_message(role, sentence["value"])
+            conv.append_message(
+                role, replace_special_tokens(tokenizer, sentence["value"])
```

Comment:
@Trangle why does the input data contain BOS/EOS token? Should you clean it in your data-cleaning scripts?

Reply:
Move this to your data cleaning code.

Reply:
> @Trangle why does the input data contain BOS/EOS token? Should you clean it in your data-cleaning scripts?

OK